dlt/extract/items.py

```diff
@@ -81,20 +81,20 @@ class SourcePipeItem(NamedTuple):
 
 
 class DataItemWithMeta:
-    __slots__ = "meta", "data"
+    __slots__ = ["meta", "data"]
 
-    meta: Any
-    data: TDataItems
+    # meta: Any
```

Comment:
Why are these commented? Same question for `table_name` on `TableNameMeta`.

Reply:
those are slot classes: https://wiki.python.org/moin/UsingSlots

I was using them incorrectly. this still needs a cleanup. ie. use tuples instead of lists